tools/doc/addga.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/sh
```

Comment:
let's rename to something more comprehensible.

`tools/api_reference/add_google_analytics.sh`?

- the `doc` sounds too general
- no one will know what `addga` is

Reply:
Done.